src/functions.inc.php

```diff
@@ -9,23 +9,17 @@
  * file that was distributed with this source code.
  */
 
-function defer(?SplStack &$context, callable $callback): void
+function defer(?SplStack &$context, callable $callback): SplStack
```

Comment:
I cannot approve changing the header of this function, I explained it in the previous comment:

```suggestion
function defer(?SplStack &$context, callable $callback): void
```